### ISSUE TITLE
Permission denied error when using Singularity container

### ISSUE THREAD

**Issue by boasvdp** [Gabaldonlab/jloh#13]
Thanks for providing a Docker container for this software! I have been using the Docker container on my laptop and it works great. 

I also wanted to use this in a Singularity container on our HPC, where I only have access to Singularity. However, I ran into some issues:

A symlink to `/root/src/jloh/jloh` is located in `/usr/bin/`, but even so the Sing container cannot find `jloh` in `$PATH`, while `/usr/bin` is listed there:
```
[puttevdb@rivm-biohn-l05p test_jloh]$ singularity exec jloh_latest.sif jloh --help
FATAL:   "jloh": executable file not found in $PATH
[puttevdb@rivm-biohn-l05p test_jloh]$ singularity exec jloh_latest.sif env | grep ^PATH
PATH=/usr/local/bin:/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin
```

`file` indicates the symlink might be broken:
```
[puttevdb@rivm-biohn-l05p test_jloh]$ singularity exec jloh_latest.sif file /usr/bin/jloh
/usr/bin/jloh: broken symbolic link to /root/src/jloh/jloh
```

If I point directly to `/usr/bin/jloh` I get a permission denied error while executables such as `awk` in the same directory do not give this error:
```
[puttevdb@rivm-biohn-l05p test_jloh]$ singularity exec jloh_latest.sif /usr/bin/jloh --help
FATAL:   stat /usr/bin/jloh: permission denied
[puttevdb@rivm-biohn-l05p test_jloh]$ singularity exec jloh_latest.sif /usr/bin/awk -V
GNU Awk 4.2.1, API: 2.0 (GNU MPFR 4.0.2, GNU MP 6.1.2)
Copyright (C) 1989, 1991-2018 Free Software Foundation.
<etc>
```

This PR by @DininduSenanayake would hopefully solve this: https://github.com/Gabaldonlab/jloh/pull/12

**Comment by DininduSenanayake** [comment]
@boasvdp While we wait for the developers to merge the PR,  I have uploaded a Docker image with the fix to my registry (https://hub.docker.com/r/dinindusenanayake/jloh/tags). 

```bash
$ singularity pull jloh.simg docker://dinindusenanayake/jloh:v1.02                                                                                                                                            
```
```bash
$ singularity shell jloh.simg
Apptainer> jloh --help


    JLOH 
    Matteo Schiavinato
    Barcelona Supercomputing Center (BSC) 
    2023 
    contact: matteo.schiavinato.90@gmail.com

    v1.0.2

    ####

 -- Extraction
    stats               Estimate heterozygous and homozygous SNP statistics
````

**Comment by MatteoSchiavinato** [comment]
Hi guys,

You caught us in a moment where the maintenance is being passed on between people. It may take a while, but it's in the todo list!

Sorry,
Matteo

**Comment by boasvdp** [comment]
No worries, appreciate that you're planning to solve this! For now I'm happy using the container made by @DininduSenanayake

**Comment by MatteoSchiavinato** [comment]
Fixed #12 in https://github.com/Gabaldonlab/jloh/commit/4753184f161b45aecb907033f0522be5ded186c9.

**Comment by RhettRautsaw** [comment]
FYI, your latest releases v1.0.3 and v1.1.0 are not on dockerhub as far as I can tell. And when I pull the latest from cgenomics/jloh and attempt to run a help command, it tells me that jloh is not in the PATH

```
> singularity exec -B $PWD:$PWD jloh_latest.sif jloh -h
FATAL:   "jloh": executable file not found in $PATH
```

**Comment by heloint** [comment]
Hi @RhettRautsaw, thank you for pointing it out.
You can find the fix in e716eb2 .
 Also, note that the correct (and the latest one) version is already the v1.1.1 with the patch.
